### ISSUE TITLE
'Ghosted pet action' tracking

### ISSUE THREAD

**Issue by ramram1048** [ramram1048/kagami#8]
### Ghosted action🤔?
Sometime pets(egis, fairies, bahamut, pheonix, seraphim) are went wrong and do no action they ordered. Maybe *kagami* can check those actions and show how much actions are lossed.

**Comment by ramram1048** [comment]
solved in #12, but some bugs remained.

### Log parsing flow:
1. listen pet action log → show action icon
2. subscribe action by `hit ID`
3. listen `hit ID` log → unsubscribe action
OR listen unsummoned log before listen hit log → check subscribed actions as `ghosted`

### Parse examples:
0. player enkindle → pet action → show action → damage dealt and gain ruin iv buff (normal)
1. player enkindle → pet action → show action → damage dealt and gain ruin iv buff → unsummoned (normal)
2. player enkindle → pet action → show action → damage dealt but ruin iv buff missing → unsummoned (check icon as `ghosted`)
3. player enkindle → pet action → show action → no-damage dealt → unsummoned (show action as `ghosted`)
4. player enkindle → unsummoned (no action shown)

### Remained bug:
# TLDR: Bug with low-level striking dummy battle. 

if egi action dealt more than dummy's max HP, no hit log created from FFXIV. so we cannot listen `hit ID` log and unsubscribe action, so **all pet actions checked as `ghosted`** when pet unsummoned.
